### ISSUE TITLE
KeyError/ValueError

### ISSUE THREAD

**Issue by myangelaku** [circleguard/circleguard#107]
```
[DEBUG] 2020/12/09 18:39:21.0882 Loading ReplayMap(timestamp=2020-11-01 01:50:40,map_id=1583472,user_id=17979769,mods=HDDT,cache=None,replay_id=3303128726,loaded=True,username=bruhnimal) (circleguard.loadables.ReplayMap, loadables.py:907)
[DEBUG] 2020/12/09 18:39:21.0883 Loaded ReplayMap by 17979769 on 1583472 already loaded, not loading (circleguard.loadables.ReplayMap, loadables.py:909)
[ERROR] 2020/12/09 18:39:22.0380 Error while running circlecore. Please report this to the developers through discord or github.
 (circleguard_gui, main_tab.py:503)
Traceback (most recent call last):
  File "site-packages\slider\library.py", line 256, in lookup_by_id
  File "site-packages\slider\library.py", line 224, in _raw_read_beatmap
KeyError: 1583472

During handling of the above exception, another exception occurred:

Traceback (most recent call last):
  File "site-packages\slider\beatmap.py", line 154, in parse
ValueError: not enough values to unpack (expected at least 2, got 1)

During handling of the above exception, another exception occurred:

Traceback (most recent call last):
  File "gui\main_tab.py", line 421, in run_circleguard
  File "site-packages\circleguard\circleguard.py", line 211, in ur
  File "site-packages\circleguard\loadables.py", line 652, in beatmap
  File "site-packages\slider\library.py", line 260, in lookup_by_id
  File "site-packages\slider\library.py", line 408, in download
  File "site-packages\slider\beatmap.py", line 1905, in parse
  File "site-packages\slider\beatmap.py", line 157, in parse
ValueError: failed to parse TimingPoint from ' '
```

**Comment by tybug** [comment]
looks like another instance of a manually edited beatmap which slider fails to parse. Is fixed by https://github.com/llllllllll/slider/issues/71

**Comment by tybug** [comment]
fixed in 2.12.2
